packages/zod-openapi/src/index.ts

```diff
@@ -32,6 +32,10 @@ import { mergePath } from 'hono/utils/url'
 import type { AnyZodObject, ZodSchema, ZodError } from 'zod'
 import { z, ZodType } from 'zod'
 
+type RouteConfig = RouteConfigBase & {
+  middlewares?: MiddlewareHandler[]
```

Comment:
`middlewares` is not bad, but we have also used `middleware` to describe multiple middleware, so `middleware` is better.

Plus, I think it would be good if it allows `MiddlewareHandler | MiddlewareHandler[]`.